pkg/services/folder/folderimpl/folder.go

```diff
@@ -284,6 +287,93 @@ func (s *Service) DeleteFolder(ctx context.Context, user *user.SignedInUser, org
 	return dashFolder, nil
 }
 
+func (s *Service) Create(ctx context.Context, cmd *folder.CreateFolderCommand) (*folder.Folder, error) {
+	// check the flag, if old - do whatever did before
+	//  for new only the store
+	cmd.UID = util.GenerateShortUID()
```

Comment:
I think this should happen only if original command UID is blank (otherwise - how can I provision a folder with a given UID?)